openapi/jira/types.bal

```diff
@@ -0,0 +1,5174 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+# Properties of a workflow status.
+public type WorkflowStatusProperties record {
+    # Whether issues are editable in this status.
+    boolean issueEditable;
+};
+
+# Details about a notification associated with an event.
+public type EventNotification record {
+    # Expand options that include additional event notification details in the response.
+    string expand?;
+    # The ID of the notification.
+    int id?;
```

Comment:
Is there a chance to update fields like `id` as required fields since the `id` could be a primary key most of the times?

Reply:
I didn't test each and every function. But most probably there is no chance to update id. But since many of the required fields and optional fields are specified by their definition correctly. I didn't change in this case